### DISCUSSION TITLE
Questtion about  Error when running code

### DISCUSSION THREAD

**Discussion by zekedz123** [jyjblrd/Mocap-Drones#28] [Q&A] [answered]
hi, thanks for the amazing work on the code. however i am ran into a issue when executing the code.  index.py  works without any error. however when i try to access the website (local host) i got the following error. what might be the issue?
 
![Screenshot from 2024-02-07 21-19-41](https://github.com/jyjblrd/Mocap-Drones/assets/46879820/ccf83ca2-4755-4f57-9d65-3e4e76c1a1bd)
![Screenshot from 2024-02-07 21-20-13](https://github.com/jyjblrd/Mocap-Drones/assets/46879820/a7f207d1-3074-44c8-8c06-e59027b387bd)

**Comment by zekedz123** [comment]
![Screenshot from 2024-02-07 21-22-45](https://github.com/jyjblrd/Mocap-Drones/assets/46879820/79767bb4-55db-4730-8c90-978a6b6b5b32)

the file trajectoryplanning is there in components folder

**Comment by jyjblrd** [comment, marked as answer]
Does [this commit](https://github.com/jyjblrd/Mocap-Drones/commit/2c186c569a6504c630a22ef38eb9ad6e736626d5) fix the problem? Try pull the repo and run it again.

**Comment by zekedz123** [comment]
yes it does. thank you!

**Comment by zekedz123** [comment]
can i ask which file contain the port information for the ps3 cameras? because the code doesn't seem to recognize the camera's connected

**Reply by jyjblrd** [reply]
the connection with the cameras is managed by this library: [github.com/bensondaled/pseyepy](https://github.com/bensondaled/pseyepy). It has instructions on connecting cameras.
